src/hb-aat-layout-feat-table.hh

```diff
@@ -95,11 +161,53 @@ struct feat
 {
   static const hb_tag_t tableTag = HB_AAT_TAG_feat;
 
+  inline unsigned int get_features (unsigned int                  start_offset,
+				    unsigned int                 *count,
+				    hb_aat_layout_feature_type_t *features) const
+  {
+    unsigned int feature_count = featureNameCount;
+    if (count && *count)
+    {
+      unsigned int len = MIN (feature_count - start_offset, *count);
```

Comment:
Same re using `hb_array_t` and `.sub_array()`.

Reply:
Thought about this before, no sub_array on UnsizedArrayOf, and doesn't worth to IMO.

Reply:
UnsizedArrayOf should not be used directly.  Maybe I remove operator[] from it.  Idea is you bind it with a length to get a hb_array_t and use that, which has a sub_array().

Reply:
UnsizedArrayOf::as_array().